releaf-content/spec/features/nodes_spec.rb

```diff
@@ -159,7 +160,8 @@
           click_button "Move"
         end
 
-        expect(page).to have_css('.copy-or-move-node-dialog .form-error-box', text: "Can't be parent to itself")
+        error_text = 'Node with id 3 has error "can\'t be parent to itself" on attribute "parent_id"'
+        expect(page).to have_css('.dialog .form-error-box', text: error_text)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.